doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+
+Although the control plane is logically centralized, scaling and control latency concerns must allow the actual implementation of the control plane to be distributed when required.
+
+All VNF, CNF and SW Virtualisation acceleration as well as all specific support functionality that is programmable in the forwarding plane must be confined to the well separated sections or stages of any shared Underlay Networking. A practical example could be a SW Virtualisation instance or VNF/CNF that controls a NIC/SmartNIC where the Underlay Networking (Switch Fabric) ensures the separation in the same way as it is done for SR-IOV cases today.
+
+The nature of a shared Underlay Network that shall ensure separation and be robust is that all code in the forwarding plane and in the control plane must be under the scrutiny and life cycle management of the HW Infrastructure Layer.
+
+This also imply that programmable forwarding functions in a Programmable Networking Fabric are shared resources and by that will have to get standardised interfaces over time to be useful for multi-vendor architectures such as ETSI NFV. Example of such future extensions of shared functionality implemented by a Programmable Networking Fabric could be L3 as a Service, Firewall as a Service and Load Balancing as a Service.
+
+> **Note:** Appliance-like applications that fully own its infrastructure layers (share nothing) could manage and utilize a Programmable Networking Fabric in many ways, but that is not a Cloud implementation and falls outside the use cases for these specifications.
 
 <a name="5.1.4"></a>
-### 5.1.4 Initial Logistical Steps
-
-> _**Editor's Note:** The purpose in this section is to communicate the responsibilities of the CNTT levels, as they pertain to the networking solution, and how they relate (i.e. what goes in each bucket). The initial content below needs to be expanded with examples of material representative of the scope for each bucket, and refined. It is expected this area be enhanced to help authors in various CNTT levels understand their purview, and ultimately be deleted._
-
-As with most CNTT subsystems, responsibility for Objectives, Requirements, Guidelines, etc. is divided among the CNTT constituencies, as follows:
-- **Tech:** High-level strategy and other informative writings
-  - e.g. Executive summary, considerations, intent, vision, etc.
-- **RM:** Generic modeling and abstraction
-  - e.g. Guidelines and other information applicable to all RAs
-  - Overall theory of operation
-- **RA-x:** architecture, APIs and other service level details
-- **RI-x:** networking information for the relevant lab or POD
-  - Considerations related to differences between target architecture (RM 5.1) and current RI implementation
-  - Factors specific to supporting RC
-  - Known non-conformances
-- **RC:** Details related to qualification
-  - Implications of testbed network environment vs. production environments
+### 5.1.4 Networking Reference Model
```

Comment:
Can we move this to a separate PR please?
The title for this is to define the scope.